README.md

```diff
@@ -1,40 +1,55 @@
 
-There are multiple ways of streaming threat indicators to Azure Sentinel. You can use one of the Threat Intelligence Platform products that is integrated with Azure, you can connect a TAXII server to Azure Sentinel via a data connector or you can use direct integration with the  Microsoft Graph Security tiInicators API. 
+Unfortunately, Azure Sentinel doesn't have any automattic threat intelligence built-in. However, it does provide some options for companies to acquire threat intelligence. Azure Sentinel gives companies the ability to import threat indicators that they use.
+
+There are multiple ways of streaming threat indicators to Azure Sentinel. You can use one of the Threat Intelligence Platform products that is integrated with Azure, you can connect a TAXII server to Azure Sentinel via a Data Connector, or you can use direct integration with the Microsoft Graph Security Indicators API.
+
+Threat Intelligence can be an essential part of any company's effort towards being secure. Threat Intelligence is not about just ingesting data and putting it on a screen, it is absorbing data and preventing possible cyber attacks before they happen.
 
-I wasn't able to implement any of these towards EITR's Azure subscription. Regardless, Threat Intelligence can be an essential part of any companies effort towards being secure. Threat Intelligence is not about just ingesting data and putting it on a screen, it is absorbing data and preventing possible cyber attacks before they happen. 
 ## Conclusion
-In conclusion, Azure Sentinel can play an instrumental role in making your company secure. It will take some time to learn and understand but in the long term, Azure Sentinel can be a useful security tool for any company. 
-Another advantage Azure Sentinel has is Azure Sentinel has a Github [repository](https://github.com/Azure/Azure-Sentinel) where you can see created custom workbooks, hunting queries, notebooks, and playbooks for Azure Sentinel.
+In conclusion, Azure Sentinel can play an instrumental role in making your company secure. It will take some time to learn and understand, but in the long term Azure Sentinel can be a useful security tool for any company.
+
+Another advantage Azure Sentinel has is Azure Sentinel has a [Github repository](https://github.com/Azure/Azure-Sentinel) where you can see created custom workbooks, hunting queries, notebooks, and playbooks for Azure Sentinel.
 
-This summer I had learn about Microsoft Azure and it has made a better computer scientist.  The challenges of working with a cloud platform help made me improve with my problem-solving and decision making skills. I happy to hear to that I will being these skills I learn from EITR technologies in my future endeavors. 
```

Comment:
Before your final closing, maybe put a sentence or two summing up the main takeaways of what in particular you liked most, your general impressions of working with the software, or what will help you with your career going forward.